aten/src/THC/THCTensorRandom.cuh

```diff
@@ -96,278 +96,6 @@ __global__ void renormRowsL1(T* dist, long rows, long cols) {
   }
 }
 
-template <typename T>
```

Comment:
Can we delete `renormRowsL1` as well or is that needed somewhere else?

Reply:
Removed